src/ontology/edits/oto-shared.ttl

```diff
@@ -0,0 +1,123 @@
+@prefix : <http://w3id.org/oto/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@base <http://w3id.org/oto/> .
+
+<http://w3id.org/oto/oto-shared/> rdf:type owl:Ontology ;
+                                   owl:versionIRI <http://w3id.org/oto/develop/oto-shared.ttl> ;
```

Comment:
```suggestion
                                   owl:versionIRI <http://w3id.org/oto/production/oto-shared.ttl> ;
```
develop or production? This branch wants to commit into production

Reply:
It should be develop branch!